src/test/java/ru/spbstu/reader/VcfReaderTest.java

```diff
@@ -0,0 +1,63 @@
+package ru.spbstu.reader;
+
+import org.junit.jupiter.api.Test;
+import ru.spbstu.reader.dto.VcfRecord;
+
+import java.io.ByteArrayInputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class VcfReaderTest {
+
+    @Test
+    public void testReadVcfFile() throws IOException {
+        List<VcfRecord> result = VcfReader.read(new FileInputStream("src/test/resources/test_vcf_file.vcf"));
```

Comment:
Maybe it would b better to use
`clazz.getResourceAsStream(resourcePath)` ?

Reply:
+1. 
1. Also extract path to constant.
2. Test should compare expected objects to actual ones, not just list sizes.